SWSabres/Model/Announcement.swift

```diff
@@ -130,14 +130,14 @@ struct Announcement: ResponseJSONObjectSerializable
         }
     }
     
-    static func getAnnouncements(completionHandler: (Result<[Announcement], NSError>) -> Void)
+    static func getAnnouncements(_ completionHandler: @escaping (Result<[Announcement]>) -> Void)
     {
-        Alamofire.request(.GET, Announcement.endpoint).getPostsReponseArray { response in
+        Alamofire.request(Announcement.endpoint).getPostsReponseArray { response in
             completionHandler(response.result)
         }
     }
     
-    class Helper: NSObject, NSCoding
+    @objc(_TtCV8SWSabres12Announcement6Helper)class Helper: NSObject, NSCoding
```

Comment:
it said the serialized name was not stable and it would break unless the "swift 3" name was inlined

it wouldn't load the cached game data without it